src/main/java/project/onlinebookstore/controller/BookController.java

```diff
@@ -0,0 +1,35 @@
+package project.onlinebookstore.controller;
+
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import project.onlinebookstore.dto.BookDto;
+import project.onlinebookstore.dto.CreateBookRequestDto;
+import project.onlinebookstore.service.BookService;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/api/books")
```

Comment:
```suggestion
@RequestMapping("/books")
```

Add `server.servlet.context-path=/api` to application.properties